man/org.freedesktop.login1.xml

```diff
@@ -156,6 +156,7 @@ node /org/freedesktop/login1 {
       SeatRemoved(s seat_id,
                   o object_path);
       PrepareForShutdown(b start);
+      PrepareForShutdownWithMetadata(a{sv} metadata);
```

Comment:
hmm, tbh, I'd just leave `start` as a separate bool, and not migrate it into to a{sv}. It's too crucial, just pass that as first argument.